pupil-spa/src/app/services/feedback/feedback.service.ts

```diff
@@ -9,43 +9,37 @@ export class FeedbackService {
   constructor(private http: Http, private storageService: StorageService) {
   }
 
-  postFeedback(): Promise<any> {
-    return new Promise(async (resolve, reject) => {
-      const headers = new Headers();
-      headers.append('Content-Type', 'application/json');
-      const requestArgs = new RequestOptions({ headers: headers });
+  async postFeedback() {
+    const headers = new Headers();
+    headers.append('Content-Type', 'application/json');
+    const requestArgs = new RequestOptions({ headers: headers });
 
-      const storedFeedback = this.storageService.getItem('feedback');
-      const accessToken = this.storageService.getItem('access_token');
+    const storedFeedback = this.storageService.getItem('feedback');
+    const accessToken = this.storageService.getItem('access_token');
 
-      if (!storedFeedback || !accessToken) {
-        return reject('Missing data in local storage');
-      }
+    if (!storedFeedback || !accessToken) {
+      return false;
```

Comment:
async functions will turn a `return` into a `resolve` and a `throw` into a `reject`.  Judging by the previous code we should use `throw` here.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function